File: 5 - Coding Interview Workshops/Week7/CodeFiles/recursionNote.py
students = ["Zahra", "Moumita", "Anri", "Julien"]

def passNote (destination, location):
    if (location == len(students)):
        return "Destination not found"
    elif (destination == students[location]):
        return "Note delivered!"
    else:
        return passNote(destination, location + 1)

File: 5 - Coding Interview Workshops/Week07/CodeFiles/JavaScript/recursionNote.js
let students = ["Zahra", "Moumita", "Anri", "Julien"];

function passNote (destination, location) {
    if (location == len(students))
        return "Destination not found";
    else if (destination == students[location])
        return "Note delivered!";
    else
        return passNote(destination, location + 1);
}

